src/applications/caregivers/components/PreSubmitInfo/SignatureInput.jsx

```diff
@@ -38,47 +40,41 @@ const SignatureInput = props => {
    * signatureMatches: compare the normalizedSignature string again the desired value
    * isSignatureComplete: complete if the value & desired string match and the parent checkbox is checked
    */
-  const normalizedSignature = signature.value.replace(/ +(?= )/g, '');
-
+  const normalizedSignature = data.value.replace(/ +(?= )/g, '');
   const signatureMatches =
     normalizedSignature.toLocaleLowerCase() === fullName.toLocaleLowerCase();
-
   const isSignatureComplete = signatureMatches && isChecked;
 
-  // set blur event for the input
-  const onBlur = useCallback(
+  const handleChange = value => {
+    setSignatures(prevState => ({ ...prevState, [label]: value }));
```

Comment:
For some reason, I cannot get the `prevState` to stub out here. If anyone has any ideas on how to make this happen. I'd love to see that happen in one of the tests for this file.